public/agents.json

```diff
@@ -0,0 +1,170 @@
+[
+  {
+    "title": "Fan Segmentation Analysis",
+    "description": "Get a 360° breakdown of your fanbase and audience segments.",
+    "prompt": "",
+    "tags": ["Recommended", "Wrapped", "Analysis"],
+    "status": "active",
+    "isActive": true
```

Comment:
why do we need `isActive`?
- why not just `delete` the recipe if it is not active?
- in what condition do we want a `non-active` recipe?

Reply:
I recommend removing both
- `status`
- `isActive`

keep these:
- `title`
- `description`
- `prompt`
- `tags`